src/Test/Unit/WarmUp/UrlsPattern/CommandArgumentBuilderTest.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Test\Unit\WarmUp\UrlsPattern;
+
+use Codeception\PHPUnit\TestCase;
+use Magento\MagentoCloud\WarmUp\UrlsPattern\CommandArgumentBuilder;
+use PHPUnit\Framework\MockObject\MockObject;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @inheritDoc
+ */
+class CommandArgumentBuilderTest extends TestCase
+{
+    /**
+     * @var CommandArgumentBuilder
+     */
+    private $argumentBuilder;
+
+    /**
+     * @var LoggerInterface|MockObject
+     */
+    private $loggerMock;
+
+    /**
+     * @inheritDoc
+     */
+    protected function setUp()
+    {
+        $this->loggerMock = $this->getMockForAbstractClass(LoggerInterface::class);
+
+        $this->argumentBuilder = new CommandArgumentBuilder($this->loggerMock);
+    }
+
+    /**
+     * @param string $entity
+     * @param string $storeIds
+     * @param array $expected
+     * @dataProvider generateDataProvider
+     */
+    public function testGenerate(string $entity, string $storeIds, array $expected)
+    {
+        $this->loggerMock->expects($this->never())
+            ->method('info');
+
+        $this->assertEquals(
+            $expected,
+            $this->argumentBuilder->generate($entity, $storeIds)
+        );
+    }
+
+    /**
+     * @return array
+     */
+    public function generateDataProvider(): array
+    {
+        return [
```

Comment:
where is the case without provided storeIds?

Reply:
added